test/test_bson_corpus.py

```diff
@@ -131,8 +134,10 @@ def run_test(self):
 
             if not lossy:
                 # Make sure we can parse the legacy (default) JSON format.
+                # json_opts = json_util.LEGACY_JSON_OPTIONS.with_options(tz_aware=True)
+                json_opts = json_util.LEGACY_JSON_OPTIONS
```

Comment:
This can continue to be inline as before.

Reply:
Done